src/app/[bookId]/components/book-info.tsx

```diff
@@ -0,0 +1,77 @@
+'use client'
+
+import { Star } from 'lucide-react'
+import React from 'react'
+
+import { Book } from '@/types/book'
+
+interface BookInfoProps {
+  book: Book
+}
+
+const BookInfo = ({ book }: BookInfoProps) => {
+  return (
+    <div>
+      <div className='md:flex'>
+        <div className='md:flex-shrink-0'>
+          <img
+            className='h-full min-w-72 w-full object-cover md:w-48'
+            src={'https://placehold.co/600x400'}
+            alt={book.name}
+          />
```

Comment:
```suggestion
          <Image
            className='h-full min-w-72 w-full object-cover md:w-48'
            src={'https://placehold.co/600x400'}
            alt={book.name}
            width={100}
            height={100}
          />
```
replace img using Image

Reply:
Fixed